Name generator (#16)
code-tool/jaeger-client-symfony-bridge: README.md

      - 'healthcheck'
      - 'metrics'
```

## Name generation options

> You can specify just the suffix, if you name generator service is named as `jaeger.name.generator.*` or if you have any other naming scheme you can put the whole name into the configuration.

Reviewer: ```suggestion
You can configure custom name generators based on regular expression pattern, which will be evaluated for operation name.

Configuration for this feature looks like key-value list, where key - regexp pattern, value - custom name generator DI service id (see details below).

Name generator should implement an 'TODO' interface.
As a custom name generator you can specify a full DI service id, or just the suffix if your name generator service is named as `jaeger.name.generator.*`.
```

My suggestion can be rephrased because it may contain some issues with grammar and github have limited support for suggestion based on one line from file.

Keys are considered body of the regular expression pattern, do not put any modifiers (e.g. `/i`, `/g`) or slashes; `route` of the request or `name` of the command should match to use alternative generator.
Expressions are checked top to bottom, if no match is found, default generator will be used

Example bundle config with name generation feature:

```yaml
# config/jaeger.yaml
jaeger:
  name_generator:
    max_length: 32
    command:  

>       .* : 'controller'

Reviewer: ```suggestion
      '^app:report:.+': 'my_service_generator_alias'
      .* : 'controller'
```

One more example can be useful for clarity

    request:

>       'brand_routes_\d+': 'my_service_generator_alias'

Reviewer: Using digits in route names is rare, let's use another example?

```suggestion
      'user_routes_\w+': 'my_service_generator_alias'
```
